core/templates/dev/head/pages/teach.html

```diff
@@ -48,56 +48,51 @@
+          </p>
 
           <p>
-            How do pendulums work? Is light a particle or a wave? What is art?
-            Can you explain topics like these to a student who wants to learn
-            more? If so, you can use Oppia's unique technology to recreate 1-on-1
-            tutoring and reach students around the world by creating and sharing
-            a single piece of learning material.
+            In order to help address this, we're excited to announce the
+            <em>Teach with Oppia</em> program!
           </p>
 
           <p>
-            With Oppia, you can create <em>explorations</em> that help students
-            learn in a fun and creative way, using videos, images, and open-ended
-            questions. Students often have similar misconceptions, and Oppia
-            helps you organize these so that you can give targeted feedback to a
-            group of students at a time. By teaching with Oppia, you can
-            communicate with students around the world, understand different
-            points of view, and improve your skills in communication and empathy.
+            The <em>Teach with Oppia</em> program enables <strong>anyone</strong>
+            &mdash; especially students &mdash; to contribute to a collection
+            of free, accessible lessons on topics that learners find difficult.
+            The lessons use Oppia's unique technology to recreate 1-on-1
+            tutoring, and typically include clear, compelling and motivated
+            explanations, together with ample exercises that allow students to
+            gain proficiency with the subject. Students can then do these
+            lessons at their own pace, supported as much as possible by useful,
+            targeted feedback.
           </p>
+
+          <p>
+            Currently, participation in the program is <strong>free</strong>.
+            Participants will receive personalized feedback on the lessons they
+            create, as well as help with playtesting and marketing their
+            lessons. They'll also be able to connect with other like-minded
+            creators, as well as the Oppia development team if additional
+            functionality is needed for the lessons.
+          </p>
+
           <p>
-            Don't feel comfortable enough to teach yet? You can
-            also collaborate with other Oppia creators by sharing feedback on
-            lessons to help create a better experience for students. Whether
-            you're a K-12 educator, a graduate student, or an individual who's
-            passionate about a specific subject and wants to share your
-            knowledge, Oppia welcomes you!
+            Although the program runs year-round, we can only accommodate a
+            limited number of creators. If you'd like to take part, please fill
+            out <a href="https://goo.gl/forms/ljmQ9R3AZRkT2kU23">this form</a>,
```

Comment:
Is there a deadline for filling this out? The language suggests there may be, but none is mentioned. Will we just remove the link one day?